docs/ExpectAPI.md

```diff
@@ -1192,13 +1192,16 @@ test('this house has my desired features', () => {
 });
 ```
 
-### `.toMatchSnapshot(optionalString)`
+### `.toMatchSnapshot(propertyMatchers, snapshotName)`
```

Comment:
I think it will be better to go with something like this:
```
### `.toMatchSnapshot(?propertyMatchers|snapshotName, ?snapshotName)`
```
Even though we explain this later